demo/chat/chat.js

```diff
@@ -118,6 +118,7 @@ function acceptInput(is) {
       if (msg.startsWith("start")) {
         start(true);
       } else {
+        start(false);
```

Comment:
This actually doesn't seem correct - if the user pastes in multiple messages that aren't "start" this would result in `start(false)` being called multiple times, which would create a fresh PeerConnection that doesn't know about previously pasted descriptions or candidates. (Same for the Go side of things)